web-common/src/features/connectors/olap/constants.ts

```diff
@@ -0,0 +1,26 @@
+export const CLICKHOUSE_DEFAULTS = {
+  "rill-managed": {
+    host: { value: "", placeholder: "Managed by Rill" },
+    port: { value: "9000", placeholder: "9000" },
+    username: { value: "", placeholder: "default" },
+    password: { value: "", placeholder: "Your ClickHouse password" },
+    ssl: { value: true },
```

Comment:
If the user selects `rill-managed`, these fields will not be shown to the user. So can we remove them here?